### PR TITLE
Rewrite of evaluateTestQuestions

### PR DESCRIPTION
**Pull request by tomveldhuis**

### CODE REVIEW

**Comment by tomveldhuis** [comment]
One thing to note: on the 560-th question during the -atest, a weird print containing the system answers of that question appears in the line with current results. Currently, I do not know the cause of this, but it does not seem to interfere with the process of evaluateTestQuestions() so far.

**Comment by tomveldhuis** [comment]
Same problem seems to occur for -test. However, I can see a pattern with this behaviour: the questions that return these weird prints all have numbers in their questions (specifically year numbers).

**Comment by nvbln** [comment]
> Same problem seems to occur for -test. However, I can see a pattern with this behaviour: the questions that return these weird prints all have numbers in their questions (specifically year numbers).

I might know the reason for this in that case. I think some part of my date functionality might print something. I'll take a look at it later.

File: main.py [hunk 1]
```diff
@@ -44,82 +44,77 @@ def evaluateTestQuestions(test_choice):
...
-                    print(systemAnswer +  " vs " + correctAnswer)
-                if systemAnswer == correctAnswer:
-                    total_correct += 1
-                else:
-                    total_incorrect += 1
-            # There aren't any correct answers
-            elif len(answer) == 0:
+                
+                # Printing the current results 
+                print("\rCorrect answers: " + str(total_correct) + "/" + str(total_lines) + ", Wrong answers: " + str(total_incorrect) + "/" + str(total_lines), end="")
```
Comment (nvbln) [inline main.py]: A bit of a small concern, but could you split this statement over multiple lines? (Like I have done previously?)

**Review by nvbln — APPROVED** [review]

**Comment by tomveldhuis** [comment]
Current results of testing:
--test gives a percentage of 8.18%
--atest gives a percentage of 9.50%

File: main.py (hunk 1 above)
Comment (tomveldhuis) [reply main.py]: Done

**Pull request merged**